### ISSUE TITLE
missing file in installation?

### ISSUE THREAD

**Issue by pl52feve** [sraoss/pg_ivm#89]
Hi guys,

my postgres instance runs with all the default configurations on a Oracle Linux 7.9 that I've set up in WSL. Now when I try install the extension the 'make install' command stops at 
`make install
gcc -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -DLINUX_OOM_SCORE_ADJ=0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -fPIC -I. -I. -I/usr/include/pgsql/server -I/usr/include/pgsql/internal -D_GNU_SOURCE -I/usr/include/libxml2   -c -o createas.o createas.c
createas.c:23:34: fatal error: catalog/pg_trigger_d.h: No such file or directory
 #include "catalog/pg_trigger_d.h"
                                  ^
compilation terminated.
make: *** [createas.o] Error 1`

Do you have any idea what might cause this error?

Thank you so much in advance for your help.

**Comment by TheHett** [comment]
You need to install package `postgresql16-dev`

**Comment by yugo-n** [comment]
> You need to install package postgresql16-dev (or another version)

Yes. You need to install postgresql*-dev containing header files (*.h) which are necessary to build pg_ivm.
